src/features/customerView/createLease/queries.ts

```diff
@@ -24,3 +24,28 @@ export const CREATE_LEASE_HARBORS_QUERY = gql`
     }
   }
 `;
+
+export const CREATE_WINTER_STORAGE_LEASE_MODAL_QUERY = gql`
+  query CREATE_WINTER_STORAGE_LEASE_MODAL($customerId: ID!) {
+    profile(id: $customerId, serviceType: BERTH) {
```

Comment:
```suggestion
    profile(id: $customerId, serviceType: BERTH) {
      id
```
Sometimes Apollo cache starts acting up when the response doesn't have the id